Cargo.toml

```diff
@@ -13,6 +13,3 @@ panic = "abort"
 inherits = "test"
 opt-level = 3
 debug = false
-
-[patch.crates-io]
-ckb-sdk = { git = "https://github.com/nervosnetwork/ckb-sdk-rust" }
```

Comment:
Unrelated to the PR description

Reply:
The second line of the PR description mentioned this change
`This PR also removes mis-introduced patch section in root Cargo.toml`